stm32cube/stm32f4xx/README

```diff
@@ -52,8 +52,4 @@ Patch List:
     -Added stm32cube/stm32f4xx/drivers/include/stm32_assert.h
     -Removed unused stm32cube/stm32f4xx/drivers/include/stm32_assert_template.h
 
-   *Fix to remove PAGESIZE definition which conflicts with POSIX
-    Impacted files:
-     drivers/include/Legacy/stm32_hal_legacy.h
```

Comment:
I think this fix should be kept as `PAGESIZE` collides with POSIX.

Reply:
NB The stm32f4xx/drivers/include/Legacy/stm32_hal_legacy.h

Reply:
Yes, and actually the fix should also be kept.